packages/block-library/src/missing/test/edit-integration.native.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import { initializeEditor } from 'test/helpers';
+
+/**
+ * WordPress dependencies
+ */
+import { getBlockTypes, unregisterBlockType } from '@wordpress/blocks';
+import * as i18n from '@wordpress/i18n';
+jest.mock( '@wordpress/i18n', () => {
+	const actual = jest.requireActual( '@wordpress/i18n' );
+	return {
+		...actual,
+		_x: jest.fn( actual._x ),
+	};
+} );
+
+/**
+ * Internal dependencies
+ */
+import { registerCoreBlocks } from '../..';
+
+beforeAll( () => {
+	// Register all core blocks
+	registerCoreBlocks();
+} );
+
+afterAll( () => {
+	// Clean up registered blocks
+	getBlockTypes().forEach( ( block ) => {
+		unregisterBlockType( block.name );
+	} );
+} );
+
+describe( 'Unsupported block', () => {
+	it( 'requests translated block title', async () => {
+		const initialHtml = `<!-- wp:table -->
+		 <figure class="wp-block-table"><table><tbody><tr><td>1</td><td>2</td></tr><tr><td>3</td><td>4</td></tr></tbody></table></figure>
+		 <!-- /wp:table -->`;
+		await initializeEditor( {
+			initialHtml,
+		} );
+
+		// jest spy for the _x translation function
+		const _xSpy = jest.spyOn( i18n, '_x' );
+
+		expect( _xSpy ).toHaveBeenCalled();
```

Comment:
I think this could be improved to ensure that `_x` is indeed called for the purpose of displaying the core block's title in the unsupported block placeholder, and not called somewhere else (which would also cause this test to pass). 
I'm not sure how to do this yet but will keep looking.

Reply:
This difficulty might be related to how `_x` is [exported as a constant](https://github.com/WordPress/gutenberg/blob/b29e3eb11dfac3d19ee5fad52046c2f31822b733/packages/i18n/src/default-i18n.js#L92), because I was getting a `TypeError: Cannot redefine property jest` error here IIRC.

Reply:
Also, I'm not sure if it's worth making a separate test to verify that the title is called when the Unsupported Block placeholder is tapped and the bottom sheet is presented.

Reply:
One of the purposes of integration tests is to query elements in a similar way as the user would do when using the app, for example looking for an element by text or by the a11y label for non-text elements. Due to this reason, I think we should figure out a different approach to verify that the unsupported block title is present and translated.

One idea I'm thinking of is to query by text within the unsupported block, as an example for the Table block, we could try to query if the text `Table` is present.

On the other hand, verifying the translation can be tricky but probably we could set up a locale and include a mocked translation to verify it, wdyt?